### PR TITLE
update generator script to include new generator args

### PR DESCRIPTION
**Pull request by mellowagain**
# Description

requires https://github.com/fiberplane/fp-openapi-rust-gen/pull/22

# Checklist

Please make sure all of these are checked before merging. Please leave items
you think are non-applicable in the list, but use strike-through (`~~`) to
indicate they don't apply.

- [x] ~~The changes have been tested to be backwards compatible.~~
- [x] ~~The OpenAPI schema and generated client have been updated.~~
- [x] ~~PR for API is ready and reviewed: (please link)~~
- [x] ~~PR for Studio is ready and reviewed: (please link)~~
- [x] ~~PR for CLI is ready and reviewed: (please link)~~
- [x] ~~PR for FPD is ready and reviewed: (please link)~~
- [x] ~~The CHANGELOG(s) are updated.~~

When adding new `fiberplane-models` module:

- [x] ~~PR for fp-openapi-rust-gen is ready and reviewed: (please link)~~

When adding new operation types:

- [x] ~~PR for fiberplane-ot is ready and reviewed: (please link)~~

After merging, please merge related PRs ASAP, so others don't get blocked.

### CODE REVIEW

File: scripts/generate_api_client.sh [hunk 1]
```diff
@@ -20,27 +22,28 @@ if [[ -d "$SCRIPT_DIR/../../fp-openapi-rust-gen/.git" ]]; then
...
-  echo "if this fails, please ensure you have executed 'docker login' with the 'fiberplane' account (creds in 1password)"
-
   docker run --rm --pull=always \
     -v "$(dirname $SCRIPT_DIR):/local" \
     -u "$(id -u ${USER}):$(id -g ${USER})" \
     fiberplane/fp-openapi-rust-gen:latest \
     --output /local/fiberplane-api-client \
     /local/schemas/openapi_v1.yml \
     --license $LICENSE \
+    --description "$DESCRIPTION" \
```
Comment (arendjr) [inline scripts/generate_api_client.sh:66]: Should we also pass `--readme`?

Comment (mellowagain) [reply scripts/generate_api_client.sh:66]: done

File: fiberplane-api-client/Cargo.toml [hunk 2]
```diff
@@ -35,12 +35,16 @@ path = "src/lib.rs"
 required-features = []
 
 [package]
+description = "Generated API client for Fiberplane API"
 edition = "2021"
 name = "fiberplane-api-client"
-description = "Generated API client for Fiberplane API"
+readme = "README.md"
```
Comment (arendjr) [inline fiberplane-api-client/Cargo.toml:41]: Shouldn't the README this references be in the repo too?

Comment (mellowagain) [reply fiberplane-api-client/Cargo.toml:41]: oh yea my bad

**Review by arendjr — APPROVED** [review]

**Pull request merged**